src/faq.html

```diff
@@ -61,6 +61,18 @@
           <li>Reboot your system.</li>
           <li>Try again.</li>
         </ol> </p>
+  - title: >
+      I'm trying to add a Zigbee device using ZHA, but Home Assistant doesn't find it. What can I do?
+    content: >
+      <p>
+        <ol>
+          <li>Check if your device <a href="https://zigbee.blakadder.com">supports ZHA</a>.</li>
```

Comment:
Perhaps it would be best to move the user-contributed, third-party device support database into a separate FAQ section about incompletely supported devices? If a device doesn't want to join (or can't join), this isn't a matter of device support.

---

Device joining in Zigbee kind of works like WiFi: the Zigbee stick doesn't search for devices, it instead flags the network as being "open for joins" and the device itself scans for all nearby networks after it is reset (so the device searches for Home Assistant, not the other way around). The device then picks a network and connects to it.

This is a somewhat pedantic distinction but I think it's useful because no matter what, even if a device is _completely_ unsupported software-wise, it will always join your network and at least show up. Because of this, there's nothing special ZHA can do to allow a device join beyond what it currently does: if a device doesn't want to join, it's probably not being reset properly (e.g. the timing required for some bulbs is unnecessarily precise), or there are interference issues preventing communication between the coordinator and the newly-joining device.

Now, if a device doesn't *work* properly after joining, that's definitely an issue that can be solved by either purchasing community-vetted devices (e.g. from that database, though ZHA will support any new standards-compliant devices that will not be immediately listed in it, if at all), or contributing [a quirk](https://github.com/zigpy/zha-device-handlers/) to help ZHA interpret the behavior of strange devices.

Reply:
Thank you for the super quick feedback, @puddly! And for the explanation. I removed the item.